### ISSUE TITLE
Feature request: delayed response

### ISSUE THREAD

**Issue by vesse** [jamonkko/console-log-server#5]
It would be useful if one could configure the server to wait for specified amount of time before sending the response.

**Comment by jamonkko** [comment]
Good idea 👍 I could add something like `--response-delay=5000` with short `-t 5000` notation to specify how many milliseconds to wait. 
To improve the usability I might need to change it log the request as soon as possible when req body has been received and log the response (if you have response logging enabled) when actually responding after the delay.

A new feature every 2 years keeps module fresh :)
